core/src/services/azfile/core.rs

```diff
@@ -421,6 +421,11 @@ impl AzfileCore {
             dirs.push_front(p);
         }
         for dir in dirs {
+            let resp = self.azfile_get_path_properties(dir).await?;
```

Comment:
The order of pathes is crucial. Implementing this way won't achieve our desired goal.

Reply:
You are right, check dir does not exist maybe from `a/b/c` to `a` is better